test/items_test.rb

```diff
@@ -0,0 +1,38 @@
+require 'time'
+require 'bigdecimal'
+require 'minitest/autorun'
+require 'minitest/pride'
+require './lib/items'
+
+
+class ItemTest < Minitest::Test
+
+  def setup
+    @data = Item.new({
+      :id => 1,
+      :name => "Pencil",
+      :description => "You can use it to write things",
+      :unit_price => BigDecimal.new(10.99,4),
```

Comment:
There is a lot of tricky formatting for Items,  not quite sure how to use BigDecimal properly.